apps/web/src/components/Basenames/RegistrationFlow.tsx

```diff
@@ -115,7 +115,7 @@ export function RegistrationFlow() {
 
   return (
     <>
-      {isDevelopment && <RegistrationStateSwitcherDynamic />}
+      {false && isDevelopment && <RegistrationStateSwitcherDynamic />}
```

Comment:
Deliberate?

Reply:
yeah, imo this should be opt-in if you need it, and was messing with the menu I was working on here